src/api_routes.rs

```diff
@@ -38,6 +39,15 @@ pub fn config(cfg: &mut web::ServiceConfig, rate_limit: &RateLimit) {
           .wrap(rate_limit.search())
           .route(web::get().to(route_get::<Search>)),
       )
+      .service(
+        web::scope("/tagline")
+          .wrap(rate_limit.message())
+          .route("", web::get().to(route_get::<GetRandomTagline>))
+          .route("", web::post().to(route_post_crud::<CreateTagline>))
```

Comment:
The GetRandom is unecessary because it can come back with GetSiteRespones.

Create, Edit, and Delete are also unecessary, as it should be part of `EditSite` and `CreateSite`, as a simple `Option<Vec<String>>`, where `Some(Vec[])` clears them, `None` doesn't change them, and `Some(Vec[tagline1, tagline2])` is an update.

Since this is a local_site type setting, it makes sense to include it in EditSite, rather than have tons of endpoints for every setting that can be updated.